.travis.yml

```diff
@@ -33,10 +34,13 @@ before_install:
       brew install yajl;
       brew install augeas --HEAD;
       brew install dbus;
-      brew install qt5;
+      # Building Qt5 currently takes too much time.
```

Comment:
> After the download finishes I will check if we can use the disk image to install Qt in the Travis build.

Please do not waste too much time on this, I made dbus optional: 4bf21f4d400de168e9e8e3edf969ade79f600e8d

A homebrew formula would be much more wanted ;)

> In theory yes

This seems to be similar to https://stackoverflow.com/questions/29587581/how-to-build-macosx-executables-on-travis-ci